zproject/prod_settings_template.py

```diff
@@ -166,21 +166,20 @@
 # directly on the Zulip server.  If file storage in Amazon S3 is
 # desired, you can configure that as follows:
 #
-# (1) Set s3_key and s3_secret_key in /etc/zulip/zulip-secrets.conf to
+# Set s3_key and s3_secret_key in /etc/zulip/zulip-secrets.conf to
 # be the S3 access and secret keys that you want to use, and setting
 # the S3_AUTH_UPLOADS_BUCKET and S3_AVATAR_BUCKET to be the S3 buckets
 # you've created to store file uploads and user avatars, respectively.
 # Then restart Zulip (scripts/restart-zulip).
-#
-# (2) Edit /etc/nginx/sites-available/zulip-enterprise to comment out
-# the nginx configuration for /user_uploads and /user_avatars (see
-# https://github.com/zulip/zulip/issues/291 for discussion of a better
-# solution that won't be automatically reverted by the Zulip upgrade
-# script), and then restart nginx.
 LOCAL_UPLOADS_DIR = "/home/zulip/uploads"
 #S3_AUTH_UPLOADS_BUCKET = ""
 #S3_AVATAR_BUCKET = ""
 
+# Setup the version of NGINX which is being used for this installation.
+# By default a NGINX version greater that 1.5.9 is assumed but if you are using
+# a older version, it is necessary to specify it here.
+# NGINX_VERSION = '1.4.6'
```

Comment:
@timabbott I was worried about the case where nginx server is separate from Django server (maybe dockers is right example) then we might end up checking versions in wrong place where Nginx if installed might be different from actual Nginx serving the files.
What are your thoughts, should we just go ahead with checking versions as this might not a case that Zulip should worry about? (Having different servers for Nginx and Django also increase scope for scalability right?)
(Above example is same as what I initially did in PR for Django-sendfile and they suggested dockers as an example where this might be a problem)